www-client/torbrowser-bin/torbrowser-bin-14.5.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Official Tor Browser"
+
+HOMEPAGE="https://www.torproject.org"
+
+TOR_SRC_BASE_URI="https://www.torproject.org/dist/torbrowser/"
+
+SRC_URI="${TOR_SRC_BASE_URI}/${PV}/tor-browser-linux-x86_64-${PV}.tar.xz"
+inherit desktop
+
+LICENSE="MPL-2.0 GPL-2 LGPL-2.1"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+PREFIX=/opt/torbrowser-bin
+
+RDEPEND="
+	app-accessibility/at-spi2-core
+	dev-libs/glib
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	sys-apps/dbus
+	sys-libs/glibc
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+
+	x11-libs/libxcb
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXrandr
+	x11-libs/pango
+"
+
+src_unpack() {
+	unpack tor-browser-linux-x86_64-${PV}.tar.xz
+	mv "${WORKDIR}"/tor-browser/Browser "${S}"
+}
+
+src_install() {
+	insinto "${PREFIX}"
```

Comment:
IMO, as Eli remarked on IRC, using PREFIX is kind of confusing and we should find another name.